src/Pages/Detail/Aside.js

```diff
@@ -0,0 +1,289 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import styled, { css } from "styled-components";
+import { useHistory } from "react-router-dom";
+import {
+  AiOutlineHeart,
+  AiOutlineYoutube,
+  AiFillGift,
+  AiOutlineUser,
+  AiOutlineLike,
+  AiFillHeart,
+} from "react-icons/ai";
+import Button from "./Components/Button";
+import { Hr, API, Tooltip } from "./Components/Utils";
+
+const Aside = () => {
+  const [category, setCategory] = useState("");
+  const [creatorName, setCreatorName] = useState("");
+  const [classTitle, setClassTitle] = useState("");
+  const [price, setPrice] = useState("");
+  const [heart, setHeart] = useState(0);
+  const [heartToggle, setHeartToggle] = useState(false);
```

Comment:
```suggestion
  const [heartCount, setHeartCount] = useState(0);
  const [isHeartActive, setIsHeartActive] = useState(0);
```
heart와 heartToggle의 이름만으로는 구분하기 쉽지 않은 것 같습니다.
자료형에 맞는 보다 구체적인 이름을 사용해주세요